docs/core/install/windows.md

```diff
@@ -388,6 +388,40 @@ Microsoft provides images that are tailored for specific scenarios. For example,
 
 For more information about using .NET in a Docker container, see [Introduction to .NET and Docker](../docker/introduction.md) and [Samples](https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md).
 
+## Troubleshooting
+
+After installing .NET, you may run into problems trying to run .NET. This section collects those common problems and provides solutions.
+
+- [Can't find/run/locate .NET after installing](#cant-findrunlocate-net-after-installing)
+
+### Can't find/run/locate .NET after installing
+
+Most likely you've installed both the x86 (32-bit) and x64 (64-bit) versions of .NET. This is causing a conflict because when you run the `dotnet` command it's resolving to the x86 version when it should resolve to the x64 version. This is usually fixed by adjusting the `%PATH%` variable to resolve the x64 version first.
+
+01. Verify that you have both versions installed by running the `where.exe dotnet` command, you should see the x86 version listed first:
```

Comment:
```suggestion
01. Verify that you have both versions installed by running the `where.exe dotnet` command. If you do, you should see your machine architecture listed first. This issue happens when the other version is listed first like in the following example running on a x64 machine:
```

Reply:
This isn't exactly true because it doesn't show a 64 folder to indicate the architecture like it does with 86. How about
```suggestion
01. Verify that you have both versions installed by running the `where.exe dotnet` command. If you do, you should see an entry for both the _Program Files\\_ and _Program Files (x86)\\_ folders. If the _Program Files (x86)\\_ folder is first as indicated by the following example, it's incorrect and you should continue on to the next step.
```